lib/clusterable/ht_item.rb

```diff
@@ -37,20 +37,12 @@ def ic_volumes
         end
       end
 
-      def pd_volumes
```

Comment:
We don't currently need to iterate over pd or all volumes, but we can add this back if we need it.